test/Api.IntegrationTest/SecretsManager/Controllers/ProjectsControllerTests.cs

```diff
@@ -295,11 +295,7 @@ public async Task Get_SmAccessDenied_NotFound(bool useSecrets, bool accessSecret
             Name = _mockEncryptedString,
         });
 
-        var mockEncryptedString2 =
-            "2.3Uk+WNBIoU5xzmVFNcoWzz==|1MsPIYuRfdOHfu/0uY6H2Q==|/98xy4wb6pHP1VTZ9JcNCYgQjEUMFPlqJgCwRk1YXKg=";
-        var request = new ProjectCreateRequestModel { Name = mockEncryptedString2 };
-
-        var response = await _client.PutAsJsonAsync($"/projects/{project.Id}", request);
+        var response = await _client.GetAsync($"/projects/{project.Id}");
```

Comment:
Accidentally found broken test when doing research on how other integration tests are written.
On similar topic i think we should have a way to distinguish in tests whether we got 404, because the api path does not exist vs explicitly returning 404 as response.

Reply:
Good find!

I'm not sure of a way, but it would be useful to be able to differentiate between the two.